webapp/src/main/webapp/src/app/groups/groups.component.ts

```diff
@@ -46,7 +46,7 @@ export class GroupsComponent implements OnInit, OnDestroy {
     }
 
     this.schoolDropdownOptions = this.filterOptions.schools.map(school => <any>{
-      label: school.name,
+      label: `${school.name} (${school.naturalId})`,
```

Comment:
Wonder if we should de-emphasize the natural ID in the parens by making it gray. Or maybe it's best to keep it black since it's searchable. Not sure.

Reply:
I can definitely de-emphasize and keep the natural id searchable using an ngTemplate.

@aaronsleeper @esotrope  what do you think?

![image](https://user-images.githubusercontent.com/5771116/30285008-8938f506-96d1-11e7-861e-00e459cd5d08.png)

Reply:
The word wrap looks pretty awful but real school IDs probably aren't going to be that long i guess. Other than that I think it looks a little nicer.

Reply:
will users know the NaturalID?
if yes, and it's necessary, then I think it should be on a second line and have .small applied to de-emphasize it.

Reply:
Can do.  Should I still keep it in parens?

Reply:
nah. not necessary

Reply:
![image](https://user-images.githubusercontent.com/5771116/30342525-0f048b40-97af-11e7-874e-1c5d21668b50.png)